src/style.css

```diff
@@ -1 +1,74 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/AvenirLTStd-Book.woff2) format('woff2');
+  font-weight: 400;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+}
+
+input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  color: #3D4E61;
+}
+
+
+
+.input {
+  background-image: url(images/Icon.png);
+  background-repeat: no-repeat;
+  background-position: 5px;
+  padding-left: 30px;
+  border-radius: 3px;
+  border: 1px solid #E1E7ED;
+  font-size: 16px;
+}
+
+.input_item-1 {
+  width: 900px;
+  height: 70px;
+  top: 71px;
+  left: 48px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+
+}
+
+.input_item-2 {
+  width: 427px;
+  height: 42px;
+  top: 403px;
+  left: 48px;
```

Comment:
Similar to the previous comment, the 'top' and 'left' properties will not affect the element unless it has a specified position. Consider removing these properties or setting a position for the element.